specificatie/openapi.yaml

```diff
@@ -662,8 +640,6 @@ paths:
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.1.0/api-specificatie/common.yaml#/components/responses/503"
         'default':
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.1.0/api-specificatie/common.yaml#/components/responses/default"
-      tags:
-      - Historie
 components:
   parameters:
     burgerservicenummer:
```

Comment:
door deze constructie zit in de property burgerservicenummer (in het antwoord) ook de specificatie met pattern, minLength en maxLength, terwijl die alleen voor de parameter nodig is.

Reply:
Issue voor aangemaakt. Zie issue #661